docker-compose.yml

```diff
@@ -15,10 +15,11 @@ x-airflow-common: &airflow-common
     - catalog/.env
   build:
     context: ./catalog/
-    args:
+    target: cat
+    args: # Automatically inferred from env vars, unless specified
       - REQUIREMENTS_FILE=requirements_dev.txt
-      - PROJECT_PY_VERSION=${PROJECT_PY_VERSION}
-      - PROJECT_AIRFLOW_VERSION=${PROJECT_AIRFLOW_VERSION}
+      - CATALOG_PY_VERSION
```

Comment:
Nice! That's really useful syntax

Reply:
I was surprised to know this too! I had always written `env` as a dict before.